easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.3-foss-2018a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'MakeCp'
+
+name = 'AUGUSTUS'
+version = '3.3'
+
+homepage = 'http://bioinf.uni-greifswald.de/augustus/'
+description = "AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences"
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+sources = ['augustus-%(version)s.tar.gz']
+source_urls = ['http://bioinf.uni-greifswald.de/augustus/binaries/']
+
+files_to_copy = ['bin', 'include']
+
+patches = [
+    '%(name)s-%(version)s_config.patch',
+    '%(name)s-%(version)s_bamtools_includepath.patch',
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Boost', '1.66.0'),
+    ('GSL', '2.4'),
+    ('lpsolve', '5.5.2.5'),
+    ('SuiteSparse', '5.1.2', '-METIS-5.1.0'),
+    ('BamTools', '2.5.1'),
+    ('SQLite', '3.21.0'),
+]
+
+files_to_copy = ['bin', 'include']
```

Comment:
Is there a specific reason why the `config` and `scripts` directories are no longer being included?

Reply:
No reason. I'll add them